impl/src/main/java/org/jboss/cdi/tck/tests/event/metadata/EventMetadataTest.java

```diff
@@ -67,14 +67,8 @@ public void testSimpleEvent(SimpleEventNotifier notifier, SimpleEventObserver ob
         notifier.fireSimpleEvent();
         verifyMetadata(observer.getSyncMetadata(), true, SimpleEvent.class, Any.class);
 
-        notifier.fireSimpleEventBeanManager();
-        verifyMetadata(observer.getSyncMetadata(), false, SimpleEvent.class, Any.class);
-
         notifier.fireSimpleEventWithQualifiers();
         verifyMetadata(observer.getSyncMetadata(), true, SimpleEvent.class, Alpha.class, Bravo.class, Any.class);
-
-        notifier.fireSimpleEventBeanManagerWithQualifiers();
-        verifyMetadata(observer.getSyncMetadata(), false, SimpleEvent.class, Charlie.class, Any.class);
     }
```

Comment:
Why not test `EventMetadata` when firing the event through `BeanManager.getEvent()`? I get it, it wasn't tested before :-)

Reply:
Well, I didn't want to introduce new assertions, just adapt existing tests and remove those that test now removed behavior.
That being said, I am not against it. Although I am not sure you can expect anything else than a `non-null` value for injection point (which is what the test asserts on). So if you want to contribute that, feel free to add it on top of this PR or even in a separate one - whatever floats your boat :-)

Reply:
I'll try that when this PR is merged :-)